src/packages/@ncigdc/components/Header.js

```diff
@@ -22,10 +22,15 @@ import { setModal } from "@ncigdc/dux/modal";
 import SessionExpiredModal from "@ncigdc/components/Modals/SessionExpiredModal";
 
 import Banner from "@ncigdc/uikit/Banner";
+import activeTheme from "@ncigdc/theme/versions/active";
```

Comment:
I don't think we should be requiring the active theme directly.
but I also don't think we need to worry about theme not being initialized. 
if we move `setTheme("active");` inside theme and maybe use an env variable for the theme name then we can just export theme and directly use it.